s3.py

```diff
@@ -237,65 +238,49 @@ def final_mirror_url(self, bucket, key):
         template = templates.get(self.url_transform, default)
         return template % dict(bucket=bucket, key=self.key_join(key))
 
-    def mirror_one(self, representation):
-        """Mirror a single representation."""
-        return self.mirror_batch([representation])
-
-    def mirror_batch(self, representations):
```

Comment:
mirror_batch already became useless when we switched to boto3, which doesn't support simultaneous uploads.